resources/profiles/Creality/filament/Creality Generic PLA High Speed @Ender-3V3-all.json

```diff
@@ -7,8 +7,5 @@
     "inherits": "Creality Generic PLA @Ender-3V3-all",
     "filament_max_volumetric_speed": [
         "23"
-    ],
-    "compatible_printers": [
```

Comment:
Due to some code issue, we need to specify compatible_printers in all instantiatiable profiles for now.
So, for now let's maintain/update this property